frontend/public/components/start-guide.jsx

```diff
@@ -30,47 +31,61 @@ class StartGuide_ extends SafetyFirst {
   }
 
   render () {
-    const openshiftFlag = this.props.flags.OPENSHIFT;
-    if (flagPending(openshiftFlag) || openshiftFlag || !this.state.visible) {
+    const { style, flags } = this.props;
+    const { OPENSHIFT: openshiftFlag, PROJECTS_AVAILABLE: projectsFlag } = flags;
+    const { visible, dismissible } = this.state;
+    const isOpenShift = !flagPending(openshiftFlag) && openshiftFlag;
+    const projectsAreAvailable = !flagPending(projectsFlag) && projectsFlag;
```

Comment:
You should be able to use `openshiftFlag` and `projectsFlag` directly below since you return early on the next line when either flag is pending.